src/sanitize.ts

```diff
@@ -239,8 +260,12 @@ export const sanitizeHTML = (text: string) => {
 	// Some texts from Agentic AI starts with a </\w+ closing tag which doesn't go through the hooks. Dompurify will remove them.
 	// The following will avoid is a fallback
 	if (text?.startsWith("</")) return text.replace("<", "&lt;").replace(">", "&gt;");
+	const configToUse =
+		customAllowedHtmlTags && customAllowedHtmlTags.length > 0
```

Comment:
How do I define a custom config with 0 allowed tags? Current logic prohibits it.

Maybe we should better use a `null` value for the `customAllowedHtmlTags` by default to cover this case.

Reply:
Done